habitat/tasks/nav/nav.py

```diff
@@ -614,7 +664,7 @@ def reset_metric(self, *args: Any, episode, **kwargs: Any):
                     self._coordinate_min,
                     self._coordinate_max,
                     self._map_resolution,
-                )[::-1]
```

Comment:
What is the reason for getting rid of the list reversal here?

Reply:
Because, `maps.draw_path` method was changed that switching x and y isn't needed anymore. That makes `maps.draw_path` easier to use.